### PR TITLE
Wip widget tests

### PR DESCRIPTION
**Pull request by SharonStrats**
Would you be able to look at peoplePicker.test.ts.  I am trying to reach the code in the findAddressBook.then (it's the else there's no selectedGroup).  In order to do that I need to resolve kb.fetcher.nowOrWhenFetched function. I can't seem to use the existing mock in rdflib or create one of my own.  I have tried creating a jest.fn, I have tried jest.Mock and jest.spyOn.  There are lots of comments in the code to explain what test I am working on and some points about it.

### CODE REVIEW

File: __mocks__/solid-auth-client.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export default {
-  currentSession: function() {
+  currentSession: function () {
```
[the remark this answers is not in the document]
Comment (michielbdejong) [reply __mocks__/solid-auth-client.ts]: ah, see https://github.com/solid/solid-ui/issues/208

File: src/widgets/peoplePicker.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ import ns from '../ns'
 import kb from '../store'
 
 export class PeoplePicker {
-  constructor (element, typeIndex, groupPickedCb, options) {
+  constructor(element, typeIndex, groupPickedCb, options) {
```
Comment (michielbdejong) [inline src/widgets/peoplePicker.js]: Doesn't `npm run lint` complain about this change?

File: __mocks__/solid-auth-client.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (SharonStrats) [reply __mocks__/solid-auth-client.ts]: I am thinking I should create another branch solving just this issue.  I feel like I haven't been separating things enough and have done too much in this branch.

**Comment by megoth** [comment]
Closing this, as it is duplicated by https://github.com/solid/solid-ui/pull/214

**Pull request merged**